src/modules/0.0.14/aws_lambda/index.ts

```diff
@@ -0,0 +1,204 @@
+import {
+  Architecture,
+  CreateFunctionCommandInput,
+  GetFunctionResponse,
+  PackageType,
+  Runtime,
+  UpdateFunctionCodeCommandInput,
+  UpdateFunctionConfigurationCommandInput
+} from '@aws-sdk/client-lambda'
+import isEqual from 'lodash.isequal'
+
+import * as metadata from './module.json'
+import { Context, Crud2, Mapper2, Module2, } from '../../interfaces'
+import {
+  addFunctionTags,
+  AWS,
+  createFunction,
+  deleteFunction,
+  getFunction,
+  getFunctions,
+  removeFunctionTags,
+  updateFunctionCode,
+  updateFunctionConfiguration,
+} from './aws'
+import { LambdaFunction } from './entity'
+import { AwsIamModule } from '../aws_iam'
+
+const base64ToUint8Array = (base64: string) => {
+  const buf = Buffer.from(base64, 'base64');
+  const bytes = new Uint8Array(buf);
+  return bytes;
+}
+
+const lambdaFunctionMapper = async (fn: GetFunctionResponse, ctx: Context) => {
```

Comment:
I don't see `zipB64` in this function. Is that on purpose?

Reply:
yes, because we do not receive the b64 from the cloud